src/Surfnet/StepupMiddleware/ApiBundle/Tests/Configuration/Entity/ConfiguredInstitutionTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Copyright 2016 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupMiddleware\ApiBundle\Tests\Configuration\Entity;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Surfnet\Stepup\Configuration\Value\Institution;
+use Surfnet\StepupMiddleware\ApiBundle\Configuration\Entity\ConfiguredInstitution;
+
+
+class ConfiguredInstitutionTest extends TestCase
+{
+    /**
+     * @test
+     * @group entity
+     */
+    public function serialized_configured_institution_has_the_correct_keys()
```

Comment:
not really sure this is a relevant test... if it is, skip the first assertion as it does not add anything, in which case the assertion given could simply be merged into the next test (without the loop please)